src/electionguard/ballot.py

```diff
@@ -612,31 +609,34 @@ class CiphertextBallot(ElectionObjectBase, CryptoHashCheckable):
      :field object_id: A unique Ballot ID that is relevant to the external system
     """
 
-    # The `object_id` of the `BallotStyle` in the `Election` Manifest
     ballot_style: str
+    """The `object_id` of the `BallotStyle` in the `Election` Manifest"""
 
-    # The hash of the election metadata
     description_hash: ElementModQ
+    """Hash of the election metadata"""
+
+    seed_hash: ElementModQ
+    """The hash used to seed the encryption"""
 
-    # The list of contests for this ballot
     contests: List[CiphertextBallotContest]
+    """List of contests for this ballot"""
 
-    # the unique ballot tracking id for this ballot
     tracking_id: Optional[ElementModQ] = field(init=False)
+    """Unique ballot tracking id for this ballot"""
 
-    # timestamp in ticks
     timestamp: int = field(init=False)
+    """Timestamp in ticks"""
```

Comment:
""" Timestamp at which the ballot encryption is generated in tick"""?

Reply:
Got it. working on it.